application/routes/vnc.py

```diff
@@ -93,7 +93,7 @@ def generate():
             'port': ws_port,
             'credentials': credentials
         }
-        yield json.dumps(result)
+        yield jsonify(result)
```

Comment:
The response type is `application/octet-stream` here and using `jsonify` introduces unnecessary overhead with mere benefits. Consider reverting the change.